### PR TITLE
Pointwise tutorial 

### PR DESCRIPTION
**Pull request by mivaia**
# Description of the problem

- No tutorial on how to use Pointwise the new meshing software used by the group.
- Pointwise tutorial is not that useful for our uses of it

# Description of the solution

- Add Pointwise tutorial

# Future work

- Add examples

### CODE REVIEW

File: doc/source/tools/pointwise/pointwise.rst [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+    :align: center
+
+- To export the 2D mesh, select all the domains. In the **file** option, select **export** and **CAE**. Choose a name for the .msh file and press save. Your mesh should be done!
+
+.. warning::
+    The Deal.II mesh file parser cannot read the comments in the ``.msh`` file generated by Pointwise. Make sure to delete manually all the commented information in the file before launching a simulation.
+ 
+The boundary conditions identification numbers are chosen by Pointwise. The user has no control over the software's decision. The boundary condition IDs can be retrieved at the end of the ``.msh`` file generated. In the present case, the boundary IDs would correspond to 2 for the inlet, 3 for the outlet and 4 for all the walls.
+
+.. image:: images/mesh_file.png
```
Comment (OresteMarquis) [inline doc/source/tools/pointwise/pointwise.rst:250]: Why is there one Unspecified?

File: doc/source/tools/pointwise/pointwise.rst [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+The best way to create 3D **block** meshes from 2D geometries is by using extrusions. The following steps can be achieved from the final 2D geometry generated previously. 
+
+- **Set Dimension** to 3D in the **CAE** menu
+- Select all the geometry and click on **Translate** in the **Edit** and **Transform** menu.
+- Insert **0.5 0 0** in the **Offset Vector** box and press enter. Click on **ok** to leave the menu. 
+
+.. image:: images/translation.png
+    :align: center
+
+The last action should have translated the geometry from 0.5 in the x direction. This translation is needed to avoid the creation of tetrahedral cells in the center of the mesh during the rotational extrusion.
```
Comment (OresteMarquis) [inline doc/source/tools/pointwise/pointwise.rst:280]: Maybe add as a note or warning?

File: doc/source/tools/pointwise/pointwise.rst [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+- **Set Dimension** to 3D in the **CAE** menu
+- Select all the geometry and click on **Translate** in the **Edit** and **Transform** menu.
+- Insert **0.5 0 0** in the **Offset Vector** box and press enter. Click on **ok** to leave the menu. 
+
+.. image:: images/translation.png
+    :align: center
+
+The last action should have translated the geometry from 0.5 in the x direction. This translation is needed to avoid the creation of tetrahedral cells in the center of the mesh during the rotational extrusion. 
+
+- In the **Create** menu, select **Extrude** and **Rotate**
```
Comment (OresteMarquis) [inline doc/source/tools/pointwise/pointwise.rst:282]: Extrude and rotate have not been introduced before in the basics section? Is there a reason for it?

File: doc/source/tools/pointwise/pointwise.rst [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+- click **Done** after the first menu appears. Enter the desired number of cells in the **Steps** box. 50 is used in this example.
```
Comment (OresteMarquis) [inline doc/source/tools/pointwise/pointwise.rst:283]: Click capital

File: doc/source/tools/pointwise/pointwise.rst [hunk 5]
```diff
@@ -0,0 +1,335 @@
...
+    :align: center
+
+The following mesh should be obtained at the end of these steps.
+
+.. image:: images/final_mesh_3d.png
+    :align: center
+
+----
+
+filling the center
```
Comment (OresteMarquis) [inline doc/source/tools/pointwise/pointwise.rst:298]: F capital

File: doc/source/tools/pointwise/pointwise.rst [hunk 6]
```diff
@@ -0,0 +1,335 @@
...
+
+- Generate a square on the bottom of the hole of the geometry with the **2 Point Curve**. The four coordinates of the square are (0.25, 0, 0), (-0.25, 0, 0), (0, 0, 0.25) and (0, 0, -0.25)
+- Connect the edges of the square to the lowest points that appeared after the splitting of the inside connectors.
+
+The image below represents the current state of the geometry. 
+
+.. image:: images/filling1.png
+    :align: center
+
+At this point, there is a problem in the geometry. The previous separation created connectors of uneven dimensions. Hovering the mouse over the inside connectors quickly shows in the bottom right corner that some are of size 14 while some are of size 13. However, the opposing sides need to be equal in a structured mesh. Some inside, middle and outside connectors will need to be modified.
```
Comment (OresteMarquis) [inline doc/source/tools/pointwise/pointwise.rst:317]: Maybe add information or labels in the figure about inside,middle and outside? I was no sure what part of the geometry you are referring to.

**Review by OresteMarquis — COMMENTED** [review]
Good job Mikael!  I was not paying attention on the spelling and grammar mistakes. I checked if the tutorial was easy to follow by just reading it. 

First code review, tell me if I was too harsh ;)

**Review by voferreira — APPROVED** [review]
Great job, Mik! This is a lovely and thorough practical guide, and I am sure this will help everyone using pointwise. :heart:

File: doc/source/tools/pointwise/images/mesh_file.png
Comment (blaisb) [inline doc/source/tools/pointwise/images/mesh_file.png:1]: Not necessary to use a screenshot for this.You can just copy paste what you want to say. I am not a fan of screenshots of text.

File: doc/source/tools/pointwise/pointwise.rst [hunk 7]
```diff
@@ -0,0 +1,335 @@
+=====================================
+Introduction on How to Use Pointwise
+=====================================
+
+This guide provides a brief overview of the core functionalities of the Fidelity Pointwise software. The content of this tutorial is divided as follows:  
+
+- :ref:`Installation <installation>`
+- :ref:`Basics <basics>`
+- :ref:`Make your first 2D Geometry <making_2D_geo>`
```
Comment (blaisb) [inline doc/source/tools/pointwise/pointwise.rst:9]:
```suggestion
- :ref:`Making your first 2D Geometry <making_2D_geo>`
```

File: doc/source/tools/pointwise/pointwise.rst [hunk 8]
```diff
@@ -0,0 +1,335 @@
...
+- :ref:`Moving to 3D <moving_3d>`
+
+
+.. _installation: 
+
+--------------------------
+Installation
+--------------------------
+
+Fidelity Pointwise is a mesh generation proprietary software. To download it, please refer to the
```
Comment (blaisb) [inline doc/source/tools/pointwise/pointwise.rst:19]:
```suggestion
Fidelity Pointwise is a proprietary mesh generation software. To download it, please refer to the
```

File: doc/source/tools/pointwise/pointwise.rst [hunk 9]
```diff
@@ -0,0 +1,335 @@
...
+
+.. code-block:: text
+    
+    chmod +x your_file.sh
+
+Execute the installation file using the following command:
+
+.. code-block:: text
+
+    ./your_file.sh
```
[the remark this answers is not in the document]
Comment (blaisb) [reply doc/source/tools/pointwise/pointwise.rst:32]: Don't you mena $> instead of "?

**Review by blaisb — COMMENTED** [review]
I just begun, I will finish Moday or maybe tonight

**Review by blaisb — COMMENTED** [review]
I don't have additional comments, but @mivaia I will give it a read again once you have addressed the existing comments

File: doc/source/tools/pointwise/pointwise.rst [hunk 10]
```diff
@@ -0,0 +1,335 @@
...
+
+
+Toolbar
+------------------
+
+Some functionalities contained in the menu bars are easily accessible through the toolbar. Here are some of the most valuable ones.
+
+The **Two Points Curve** and the **Draw Curve** displayed below at the left can be used to create connectors. The symbol of the wrench over the domain and block on the right are shortcuts to **Assemble Domains and Blocks** quickly.
+
+.. image:: images/toolbar_create.png
```
[the remark this answers is not in the document]
Comment (mivaia) [reply doc/source/tools/pointwise/pointwise.rst:105]: The point form makes it clearer in my opinion. I don't think the labels are necessary now.

File: doc/source/tools/pointwise/pointwise.rst [hunk 11]
```diff
@@ -0,0 +1,335 @@
...
+The final geometry should look like this.
+
+.. image:: images/connectors.png
+    :align: center
+
+Few geometries are nice convex quadrilaterals. Concave shapes generally lead to low-quality meshes. It is therefore ideal to separate the geometry into trapezoidal or rectangular sections that will facilitate the meshing process. This is exactly the purpose of the last two created connectors. Separating the weird elbow into squares will improve the overall quality of the generated mesh.
+
+The subsequent step involves dividing the long connectors positioned on the top and left sides to ensure their contribution is limited to the meshing of a single subsection of the geometry.
+
+- Select the top connector and go to the option **Edit** and **Split** at the top of the page. Enter 50 in the **Percent of Length** to split the connector in half. Repeat the same process for the left connector. Without this step, the meshing process is impossible.
```
[the remark this answers is not in the document]
Comment (mivaia) [reply doc/source/tools/pointwise/pointwise.rst:184]: Yes there is. It should be explained further into the guide

File: doc/source/tools/pointwise/pointwise.rst (hunk 1 above)
Comment (mivaia) [reply doc/source/tools/pointwise/pointwise.rst:250]: We did not specify te volume conditions. The solver will just assume it is the fluid

File: doc/source/tools/pointwise/pointwise.rst (hunk 3 above)
Comment (mivaia) [reply doc/source/tools/pointwise/pointwise.rst:282]: Anybody familiar with CAD should be familiar with the extrusion concept.

**Review by voferreira — APPROVED** [review]

**Pull request merged**